pgml-sdks/python/pgml/pgml/collection.py

```diff
@@ -777,5 +777,8 @@ def vector_search(
 
         search_results = run_select_statement(conn, cte_select_statement)
         self.pool.putconn(conn)
+    
+        # Sort the list of dictionaries based on the 'score' key in descending order
+        search_results = sorted(search_results, key=lambda x: x['score'], reverse=True)
```

Comment:
That's already done in SQL above. Postgres is much better at sorting than Python.

Reply:
```
 select array(select unnest(array[1.0, 2.0, 3.0]) order by 1);
```

https://stackoverflow.com/questions/2913368/sorting-array-elements